[RU] Rewrite `Chat Console` (#5730)
ppy/osu-wiki: wiki/Chat_Console/ru.md

# Окно чата

На большинстве экранов в osu! можно нажать `F8` или кликнуть `Show Chat` в нижнем правом углу для открытия окна чата в нижней половине экрана.

![Окно чата](img/Chatconsole1.png "Окно чата")

- Список вкладок доступных каналов. Просто нажмите на вкладку, чтобы перейти на этот канал. Нажмите на кнопку с жёлтым плюсом, чтобы отобразить список каналов и присоединиться к ним.
- Цвета имён пользователей означают разные вещи.

| Цвет | Кто? |
| :-- | :-- |
| **Белый** | Вы |
| **Бледно-жёлтый** | Игроки, не поддержавшие osu! |
| **Жёлтый** | [Саппортер](/wiki/osu!supporter) |
| **Красный** | Участник [Команды Глобальной Модерации](/wiki/People/The_Team/Global_Moderation_Team) или [Команды Оценки Номинаций](/wiki/People/The_Team/Nomination_Assessment_Team) |

> | **Зелёный** | Сообщение содержит Ваше имя пользователя или слово которое Вы добавили в список подсвечиваемых в чате слов. Копия сообщения так же появится в канале `#highlight`. |

Author: В текущем переводе клиента написано `Список подсвечиваемых в чате слов (через пробел)`, может быть лучше оставить `список подсвечиваемых в чате слов`, чтобы никого не смутить?

| **Синий** | Личное сообщение |
| **Голубой** | [peppy](https://osu.ppy.sh/users/2), создатель osu! |
| **Розовый** | [BanchoBot](/wiki/BanchoBot) |

- Кликните на `Show Ticker` для отображения новых сообщений в нижней части экрана, когда окно чата скрыто.
- Кликните на `Auto-Hide`, чтобы автоматически скрыть окно чата во время игры (кроме как на интро, аутро и перерывах).
- Кликните на `Hide Chat` или нажмите `F8` снова, чтобы скрыть окно чата.

## Расширенное окно чата

На большинстве экранов в osu! можно нажать `F9` или кликнуть `Online Users` в нижнем правом углу для открытия расширенного окна чата, в дополнение к окну чата, расширенное окно чата показывает на оставшейся части экрана список пользователей, которые зашли в osu!.

![Расширенное окно чата](img/Chat_Console-Extended.png "Расшриенное окно чата")

> Все пользователи, которые вошли в osu! имеют панель пользователя, отображающуюся в расширенном окне чата. По умолчанию она отображает основную информацию (имя, общий рейтинг, ранк, точность, количество игр и аватар, если он есть). Если навести курсор на панель, она отобразит другую информацию (имя, ранк, аватар, если есть, время и часовой пояс, местоположение и страну, город, если пользователь разрешил это и что он сейчас делает).

Reviewer: ```suggestion
В расширенном окне отображаются панели всех игроков, находящихся в сети. По умолчанию панель отображает основные сведения об игроке (имя, общий рейтинг, ранк, точность, количество игр и аватар, если он есть). Если навести на неё курсор, будут видны дополнительные данные (имя, ранк, аватар, если есть, время и часовой пояс, местоположение и страна, город (при согласии пользователя) и текущая активность).
```

> - `Все` - отображает всех. `Друзья` - отображает только Ваших друзей. `Рядом` - отображает игроков в ближайших городах от вашего. `Страна` - отображает игроков из Вашей страны.

Reviewer: так, в оригинале тут другое

Author: Я знаю что в оригинале тут другое, было бы не плохо это видеть и в оригинальной статье.

- Кликните на `Auto-Hide`, чтобы автоматически скрывать окно чата во время игры (не работает во время вступления к карте, её концовки и перерывов).
- Кликните на `Hide Chat` или нажмите `F8`, чтобы скрыть окно чата.

## Расширенное окно чата

*В [osu!academy](/wiki/osu!academy) рассказали об этом в [6 эпизоде (6:52)](https://www.youtube.com/watch?v=cyYRl-a5xII) вместе с [Мультиплеером](/wiki/Multi).*

Расширенное окно чата можно открыть, нажав `F9` или кликнув по кнопке `Online Users` в правом нижнем углу экрана. 

![Расширенное окно чата](img/Chat_Console-Extended.png "Расшриенное окно чата")

В расширенном окне отображаются панели всех игроков, находящихся в сети. По умолчанию панель отображает основные сведения об игроке (имя, общий рейтинг, ранк, точность, количество игр и аватар, если он есть). Если навести на неё курсор, будут видны дополнительные данные (имя, ранк, аватар, если есть, время и часовой пояс, местоположение и страна, город (при согласии пользователя) и текущая активность).

- Друзья - отображает только Ваших друзей.
- Зафиксировать панели предотвращает перемещение панелей при входе/выходе пользователей.

> - Нажмите на вкладки сверху чтобы отсортировать пользователей по определённым критериям.
> - Нажмите на Карта мира чтобы посмотреть карту мира с отображением где находятся пользователи, которые сейчас в сети.
> - Вы можете нажать и перемещать белый прямоугольник для прокрутки, так же Вы можете это сделать с помощью колёсика мыши.
> - Пользователи без статистики на панели вошли в чат с помощью IRC клиента.

Author: ```suggestion
- Вкладки `Ник`, `Ранк` и остальные сортируют пользователей по определённым критериям.
- Вкладка `Карта мира` показывает, откуда пользователи по всему миру вышли в сеть.
- Чтобы переместиться по списку, используйте колесо мыши, или перетаскивайте белый прямоугольник справа.
- Если у пользователя нет статистики, он читает чат через IRC-клиент.
```

| Цвет панели | Описание |
| :-- | :-- |
| ![Игрок ничего не делает](img/Chat_Console-Idle.png "Игрок ничего не делает") | Тёмно-синий - Игрок в данный момент ничего не делает или просто общается в чате. |
| ![Игрок играет](img/Chat_Console-Playing.png "Игрок играет") | Серый - Игрок играет карту в одиночку. |

> | ![Игрок смотрит](img/Chat_Console-Watching.png "Игрок смотрит") | Светло-синий - Игрок смотрит повтор или наблюдает за игрой другого игрока. |
> | ![Игрок редактирует карту](img/Chat_Console-Editing.png "Игрок редактирует карту") | Красный - Игрок редактирует свою карту. |
> | ![Игрок тестирует карту](img/Chat_Console-Testing.png "Игрок тестирует карту") | Фиолетовый - Игрок тестирует (в игре) чужую карту. |
> | ![Игрок загружает карту](img/Chat_Console-Submitting.png "Игрок загружает карту") | Бирюзовый - Игрок загружает (новую или обновлённую) карту, которую он сделал. |
> | ![Игрок проверяет чужую карту](img/Chat_Console-Modding.png "Игрок проверяет чужую карту") | Зелёный - Проверяет или дорабатывает чужую карту. |
> | ![Игрок в мультиплеерной комнате](img/Chat_Console-Multiplayer.png "Игрок в мультиплеерной комнате") | Коричневый - Игрок в мультиплеерной комнате, но не играет. |
> | ![Игрок играет в мультиплеер](img/Chat_Console-Multiplaying.png "Игрок играет в мультиплеер") | Жёлтый - Игрок играет в мультиплеер. |
> | ![Игрок отошёл от клавиатуры](img/Chat_Console-Afk.png "Игрок отошёл от клавиатуры") | Чёрный - Игрок отошёл от клавиатуры (AFK) . |
> | ![Игрок вошёл через IRC](img/Chat_Console-IRC.png "Игрок вошёл через IRC") | Тёмно-синий, без содержимого - Игрок не в игре, но вошёл через IRC клиент или статистика недоступна. |

Reviewer: вообще это хорошо бы переписать на безличные предложения, но я потом сам этим займусь вместе с переделкой статьи

```suggestion
| ![Игрок смотрит реплей](img/Chat_Console-Watching.png "Игрок смотрит реплей") | Светло-синий: игрок смотрит реплей или наблюдает за другим игроком. |
| ![Игрок редактирует карту](img/Chat_Console-Editing.png "Игрок редактирует карту") | Красный: игрок редактирует свою карту. |
| ![Игрок тестирует карту](img/Chat_Console-Testing.png "Игрок тестирует карту") | Фиолетовый: игрок тестирует карту в редакторе. |
| ![Игрок загружает карту](img/Chat_Console-Submitting.png "Игрок загружает карту") | Бирюзовый: игрок загружает на сервер новую версию своей карты. |
| ![Игрок проверяет чужую карту](img/Chat_Console-Modding.png "Игрок проверяет чужую карту") | Зелёный: игрок занимается [моддингом](/wiki/Modding) чужой карты. |
| ![Игрок находится в лобби](img/Chat_Console-Multiplayer.png "Игрок находится в лобби") | Коричневый: игрок находится в комнате мультиплеера, но не играет. |
| ![Игрок соревнуется в мультиплеере](img/Chat_Console-Multiplaying.png "Игрок соревнуется в мультиплеере") | Жёлтый: игрок соревнуется в мультиплеере. |
| ![Игрок отошёл от клавиатуры](img/Chat_Console-Afk.png "Игрок отошёл от клавиатуры") | Чёрный: игрок отошёл от клавиатуры (AFK). |
| ![Игрок вошёл через IRC](img/Chat_Console-IRC.png "Игрок вошёл через IRC") | Тёмно-синий, без статистики: игрок вошёл в чат через IRC-клиент, либо его статистика ещё не прогрузилась. |
```

> Нажатие на панель пользователя открывает экран с опциями

Reviewer: ```suggestion
Нажатие на панель пользователя открывает список возможных действий.
```

> ![Экран с опциями](img/Chat_Console-Userpaneloptions.jpg "Экран с опциями")

Reviewer: ```suggestion
![Список действий, которые можно совершить с пользователем](img/Chat_Console-Userpaneloptions.jpg "Список действий, которые можно совершить с пользователем")
```

> Нажмите на цифру или кликните на панель для выбора:

Reviewer: ```suggestion
Кликните по пункту, чтобы его выбрать, или нажмите клавишу с соответствующей цифрой:
```

> 1. `Наблюдать за игрой`: Если пользователь играет карту, Вы можете посмотреть за его игрой. Ваше имя пользователя будет отображено в списке наблюдателей у этого игрока.
> 2. `Открыть профиль`: Открывает профиль этого игрока в браузере.
> 3. `Начать разговор`: Открывает вкладку чата с данным пользователем.
> 4. `Пригласить в мульти`: (Если Вы в мультиплеерной комнате) Приглашает пользователя присоединиться к комнате.
> 5. `Добавить (Удалить) из друзей`: Добавляет (Удаляет) пользователя из Вашего списка друзей.
> 6. `Отправить жалобу...`: Сообщает о плохом поведении пользователя. Не должна использоваться для чего-то другого. Вы можете пожаловаться на пользователя в игре, но так же можете сделать сделать это на сайте, например на форуме.
> 7. `Игнорировать`: Скрывает сообщения данного пользователя в Вашем окне чата.
> 8. `Отмена`: Закрывает данный экран.

Reviewer: ```suggestion
1. `Наблюдать за игрой`: если пользователь играет карту, вы можете посмотреть за его игрой. Ваш ник будет отображен в списке наблюдателей у этого игрока.
2. `Открыть профиль`: открывает профиль игрока в браузере.
3. `Начать разговор`: открывает вкладку чата с данным пользователем.
4. `Пригласить в мульти`: приглашает пользователя в комнату в мультиплеере, если вы там находитесь.
5. `Добавить (удалить) из друзей`: добавляет или удаляет пользователя из вашего списка друзей.
6. `Отправить жалобу...`: открывает форму для жалобы на поведение пользователя. Используйте этот пункт, чтобы сообщить о нарушении правил внутри игры. Пожаловаться на поведение вне игры можно из профиля игрока.
7. `Игнорировать`: скрывает сообщения данного пользователя в вашем окне чата.
8. `Отмена`: закрывает список действий.
```

## Список команд

### /help

> | Команда | Результат | Пример | Ответ BanchoBot'а |

Reviewer: ```suggestion
| Команда | Результат | Пример | Ответ BanchoBot |
```

| :-- | :-- | :-- | :-- |

> | `/addfriend [пользователь]` | Добавляет `[пользователь]` в Ваш список друзей. | `/addfriend Amigo` | You are now friends with Amigo. |
> | `/delfriend [пользователь]` | Удаляет `[пользователь]` из Вашего списка друзей. | `/delfriend Amigo` | You are no longer friends with Amigo. |

Reviewer: ```suggestion
| `/addfriend [пользователь]` | Добавить `[пользователь]` в список друзей. | `/addfriend Amigo` | You are now friends with Amigo. |
| `/delfriend [пользователь]` | Удалить `[пользователь]` из списка друзей. | `/delfriend Amigo` | You are no longer friends with Amigo. |
```

> | `/away [сообщение]` | Изменяет сообщение, отправляемое когда Вы отошли от клавиатуры (отправляется когда другие пользователи отправляют Вам личное сообщение). Оставьте `[сообщение]` пустым чтобы удалить его. | `/away I am John Smith.` | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~? BanchoBot: I am John Smith. |
> | `/bb` | Отправляет сообщение BanchoBot'у, чтобы выполнить команду, например `!stats [пользователь]` | `/bb !stats Uan` | \[15/11/12\] Stats for [Uan](https://osu.ppy.sh/users/147623): Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
> | `/chat [пользователь]` | Открывает вкладку чата с данным пользователем. | `/chat Amigo` | (Открылась вкладка чата с Amigo) |
> | `/clear` | Очищает историю чата. | `/clear` | (Очищает всё в данной вкладке) |

Reviewer: ```suggestion
| `/away [сообщение]` | Настроить сообщение для автоответа другим пользователям. Оставьте `[сообщение]` пустым, чтобы выключить автоответчик. | `/away I am John Smith.` | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~? BanchoBot: I am John Smith. |
| `/bb` | Отправить сообщение лично BanchoBot, например, `!stats [пользователь]` | `/bb !stats Uan` | \[15/11/12\] Stats for [Uan](https://osu.ppy.sh/users/147623): Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
| `/chat [пользователь]` | Открыть вкладку чата с пользователем. | `/chat Amigo` | (Открылась вкладка чата с Amigo) |
| `/clear` | Очистить историю чата. | `/clear` | (Стирает весь текст в открытой вкладке) |
```

> | `/ignore [пользователь][@chp]` | Скрывает все сообщения от указанного пользователя. Добавляя @ с буквами, c, h, и/или p, Вы можете скрыть его сообщения в чате, упоминаниях или личных сообщениях соответственно. | `/ignore Amigo@chp` | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Теперь все сообщения \[c\], все упоминания \[h\] и личные сообщения \[p\] от Amigo будут скрыты. |
> | `/j [канал]` или `/join [канал]` | Присоединение к указанному каналу | `/join #lobby` | (Открывается вкладка #lobby) |
> | `/p` или `/part` | Выход из текущего канала. | `/part` | Нет |
> | `/unignore [пользователь]` | Снова показывает сообщения от указанного пользователя. | `/unignore Amigo` | Теперь до Вас будут доходить сообщения Amigo. |
> | `/me [действие]` | Выполняет действие от третьего лица. | `/me is at home` | * John is at home |

Reviewer: ```suggestion
| `/ignore [пользователь][@chp]` | Скрывать все сообщения пользователя до перезапуска игры. Дописав после ника символ `@` и буквы `c`, `h` или `p`, вы можете скрыть сообщения в чате, упоминаниях или личной переписке соответственно. | `/ignore Amigo@chp` | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Теперь все сообщения \[c\], все упоминания \[h\] и личные сообщения \[p\] от Amigo будут скрыты. |
| `/j [канал]` или `/join [канал]` | Зайти на канал | `/join #lobby` | (Открывается вкладка `#lobby`) |
| `/p` или `/part` | Покинуть открытый канал. | `/part` | Нет |
| `/unignore [пользователь]` | Перестать игнорировать сообщения от указанного пользователя. | `/unignore Amigo` | Теперь вы будете видеть сообщения от Amigo. |
| `/me [действие]` | Выполнить действие от третьего лица. | `/me is at home` | * John is at home |
```

> | `/msg [пользователь] [сообщение]` | Отправляет личное сообщение `[пользователь]`. | `/msg Amigo I am sick at home.` | (Во вкладке с Amigo) John: I am sick at home. |
> | `/np` | Отправляет в чате карту, в которую Вы играете, слушаете или смотрите повтор. | `/np` | (Если играете) * John is playing [Peter Lambert - osu! tutorial \[Gameplay Basics\]](https://osu.ppy.sh/beatmapsets/3756#osu/22538) |
> | `/reply` или `/r` | Отвечает на последнее сообщение. | `/r Do you know any good doctor?` | (Во вкладке с Amigo) \[Предыдущие сообщения\] John: I am sick at home. Amigo: Really? John: Do you know any good doctor? |
> | `/savelog` | Сохраняет историю чата в текстовый файл. | `/savelog` | (Создана папка "Chat" в директории с osu!, в которой будет сохранена история чата) |
> | `/watch [пользователь]` | Начинает наблюдение за игрой `[пользователь]`. | `/watch Amigo` | * Started spectating Amigo. (Когда Amigo начнёт играть в карту, которая у вас есть, вы сможете наблюдать за игрой Amigo) |
> | `/nopm` | Разрешает/запрещает отправлять Вам личные сообщения пользователям не из Вашего списка друзей | `/nopm` | (Появится уведомление по середине экрана, с информацией, могут ли все пользователи отправлять личные сообщения или только друзья). |
> | `/invite [пользователь]` | Приглашает `[пользователь]` в мультиплеерную комнату по ссылке. | `/invite Nathanael` | Вы приглассили Nathanael в вашу комнату. |

Reviewer: ```suggestion
| `/msg [пользователь] [сообщение]` | Отправить личное сообщение `[пользователь]`. | `/msg Amigo I am sick at home.` | (Во вкладке с Amigo) John: I am sick at home. |
| `/np` | Отправить в чат карту, которую вы играете, слушаете, или где смотрите реплей. | `/np` | (Во время игры) * John is playing [Peter Lambert - osu! tutorial \[Gameplay Basics\]](https://osu.ppy.sh/beatmapsets/3756#osu/22538) |
| `/reply` или `/r` | Ответить на последнее полученное личное сообщение. | `/r Do you know any good doctor?` | (Во вкладке с Amigo) \[Предыдущие сообщения\] John: I am sick at home. Amigo: Really? John: Do you know any good doctor? |
| `/savelog` | Сохранить историю чата в текстовый файл. | `/savelog` | (Создана папка `Chat` в директории с osu!, в которой будет сохранена история чата) |
| `/watch [пользователь]` | Начать наблюдение за игрой `[пользователь]`. | `/watch Amigo` | * Started spectating Amigo. (Когда Amigo начнёт играть имеющуюся у вас карту, вы сможете наблюдать за его действиями) |
| `/nopm` | Разрешить или запретить пользователям не из вашего списка друзей отправлять вам сообщения | `/nopm` | (В центре экрана появится сообщение о том, кто может с вами связываться). |
| `/invite [пользователь]` | Отправить `[пользователь]` ссылку-приглашение в мультиплеер. | `/invite Nathanael` | Вы пригласили Nathanael в вашу комнату. |
```

> ## /keys

Reviewer: уровень заголовка потерял
```suggestion
### /keys
```

> | Кнопки на клавиатуре | Результат |

Reviewer: ```suggestion
| Сочетание клавиш | Действие |
```

| :-- | :-- |

> | `Page Up` / `Page Down` | Прокручивает окно чата. Для этого так же можно использовать колёсико мыши. |
> | `Tab` | Автозаполнение вводимого имени пользователя. |
> | `F8` | Показать/скрыть окно чата. |
> | `F9` | Показать/скрыть расширенное окно чата. |
> | `Ctrl` + `C` / `V` | Копирование/Вставка. |
> | `Alt` + `0` - `9` | Перейти к соответствующей вкладке. |

Reviewer: ```suggestion
| `Page Up` / `Page Down` | Пролистать окно чата (альтернатива — колесо мыши). |
| `Tab` | Дополнить ник пользователя по первым буквам или показать возможные варианты. |
| `F8` | Показать/скрыть окно чата. |
| `F9` | Показать/скрыть расширенное окно чата. |
| `Ctrl` + `C` / `V` | Скопировать текст, выделенный в окне отправки сообщения, или вставить его. |
| `Alt` + `0` - `9` | Перейти к вкладке под соответствующим номером. |
```
